container/src/main/java/com/flipkart/poseidon/api/ExceptionMapper.java

```diff
@@ -0,0 +1,11 @@
+package com.flipkart.poseidon.api;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by shrey.garg on 16/02/16.
+ */
+public interface ExceptionMapper {
+    boolean map(Throwable e, HttpServletResponse response) throws IOException;
```

Comment:
Please write comments for the class (what it is meant for, scenarios to use) as well as for the method (when it should return true, false, what it can do with response, what exactly is throwable i.e. root cause etc)